_config.yml

```diff
@@ -124,6 +124,9 @@ callouts:
     title: Warning
     color: red
 
+# Dark color scheme; activates auto-switching if not nil. Supports "dark" or a custom color scheme
+dark_color_scheme: nil
```

Comment:
This setting stops the preview of the theme docs from switching between light and dark with the media preference. Please change `nil` to `dark`.

(The settings in the theme `_config.yml` don't affect those who use JtD as a gem or remote theme, so the default behaviour of your implementation is correct: auto-switching is off when `dark_color_scheme` is `nil` or unset.)

Reply:
```suggestion
dark_color_scheme: dark
```